### DISCUSSION TITLE
export recipe properties

### DISCUSSION THREAD

**Discussion by Chrysanthemum536** [TandoorRecipes/recipes#3191] [Q&A]
when exporting and then importing between instances using default export, the recipe properties were not transferred. how to fix this?

**Comment by smilerz** [comment]
Properties and other metadata are not included in export.  If you are looking to do a full transfer the a database export/import is the proper method.

**Reply by casimpunkt** [reply]
Hello, by database export do you mean creating a dump?

**Reply by smilerz** [reply]
Yes
